atm.js

```diff
@@ -1,3 +1,150 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+ //DEPOSIT MONEY FROM CHECKING
+
+
+
+  var checkingDeposit = $("#checking .deposit")
+  var savingsDeposit = $(“#savings .deposit”)
```

Comment:
Please make sure you're using the correct quotation marks in your code. I notice the ones on this line are a little different than the previous line.

Not sure why this happens sometimes but happy to look into it for you if it continues to be an issue.